packages/universal-crossplane/universal-crossplane.patch

```diff
@@ -0,0 +1,23 @@
+diff -x '*.tgz' -x '*.lock' -uNr packages/universal-crossplane/charts-original/Chart.yaml packages/universal-crossplane/charts/Chart.yaml
+--- packages/universal-crossplane/charts-original/Chart.yaml
++++ packages/universal-crossplane/charts/Chart.yaml
+@@ -1,8 +1,11 @@
+ apiVersion: v1
+-appVersion: 1.2.1-up.3
++appVersion: 1.2.1003
+ description: Upbound Universal Crossplane allows you to connect your Crossplane-enabled cluster to Upbound Cloud.
+ home: https://upbound.io
+-icon: https://crossplane.io/images/favicon_192x192.png
++icon: https://raw.githubusercontent.com/upbound/universal-crossplane/66ce9eb2c5a0c3af8ed7d19551a2c4d743b933b9/docs/media/logo.png
++annotations:
++  catalog.cattle.io/certified: partner
++  catalog.cattle.io/release-name: universal-crossplane
+ keywords:
+ - cloud
+ - infrastructure
+@@ -31,4 +34,4 @@
+ - email: info@upbound.io
+   name: Upbound Inc.
+ name: universal-crossplane
+-version: 1.2.1-up.3
++version: 1.2.1003
```

Comment:
Workaround for https://github.com/upbound/universal-crossplane/issues/119

Reply:
Just a FYI, we append the packageVersion to the chart's version to generate the version number for the package we generate. Since your packageVersion is 00 at the moment, the generated package will have the version 1.2.1.00300 which can be confusing for users.